test/iron-a11y-announcer.html

```diff
@@ -13,42 +13,38 @@
 <head>
   <title>iron-a11y-announcer</title>
 
-  <script src="../../../@webcomponents/webcomponentsjs/webcomponents-bundle.js"></script>
-  <script src="../../../wct-browser-legacy/browser.js"></script>
-
-  <script type="module" src="../../polymer/polymer-legacy.js"></script>
-  <script type="module" src="../iron-a11y-announcer.js"></script>
+  <script src=" ../node_modules/@webcomponents/webcomponentsjs/webcomponents-bundle.js"></script>
+  <script src=" ../node_modules/wct-browser-legacy/browser.js"></script>
```

Comment:
There's an extra space in front of these URLs. I'm not sure if it matters though.